src/attrs.rs

```diff
@@ -64,7 +64,7 @@ fn is_relevant_trait(item: &TraitItem) -> bool {
 }
 
 fn is_relevant_block(block: &Block) -> bool {
-    for stmt in block.stmts.iter() {
+    for stmt in &block.stmts {
```

Comment:
Yeah, this is only viable *after* `cargo test`. Then it is `-L target/libclippy.so` (at least on my system).